Plan/src/main/java/com/djrapitops/plan/system/database/databases/sql/patches/Patch.java

```diff
@@ -46,15 +48,20 @@ public Patch(SQLDB db) {
     }
 
     public boolean hasTable(String tableName) {
-        String sql = usingMySQL ?
-                "SELECT * FROM information_schema.TABLES WHERE table_name=? AND TABLE_SCHEMA=? LIMIT 1" :
-                "SELECT tbl_name FROM sqlite_master WHERE tbl_name=?";
+        String sql;
+        if (usingH2) {
+            sql = "SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME=?";
+        } else if (usingMySQL) {
+            sql = "SELECT * FROM information_schema.TABLES WHERE table_name=? AND TABLE_SCHEMA=? LIMIT 1";
+        } else {
+            sql = "SELECT tbl_name FROM sqlite_master WHERE tbl_name=?";
+        }
 
         return query(new QueryStatement<Boolean>(sql) {
             @Override
             public void prepare(PreparedStatement statement) throws SQLException {
                 statement.setString(1, tableName);
-                if (usingMySQL) {
+                if (usingMySQL && !usingH2) {
```

Comment:
I think this combined boolean should be named `requiresSchemaName` or removed altogether by hard coding the placement of TABLE_SCHEMA in sql above

Reply:
Resolved with 0a25050567f4e856cf74125585dff0b26d795f48